components/lists.css

```diff
@@ -0,0 +1,12 @@
+.list {
+  margin: 1rem;
```

Comment:
Use global variables for spacing for properties like margin, padding, and gap to maintain consistency across the website

Reply:
will try implementing this after completing p1